src/document.h

```diff
@@ -2253,11 +2253,13 @@ struct Document {
         int i = 0;
         for (auto tagit = tags.begin(); tagit != tags.end(); ++tagit)
             if (i++ == tagno) {
-                Cell *c = selected.GetCell();
-                if (!c) return OneCell();
-                c->AddUndo(this);
-                c->text.Clear(this, selected);
-                c->text.Insert(this, tagit->first, selected);
+                selected.g->cell->AddUndo(this);
+                loopallcellssel(c, true) {

```

Comment:
when is this useful? I can't imagine wanting to fill a whole grid all with the same tag?